dirsrvtests/tests/suites/clu/dsidm_role_test.py

```diff
@@ -771,9 +771,18 @@ def test_dsidm_role_subtree_status(topology_st, create_test_managed_role):
     output = 'Entry DN: {}'.format(test_role.dn)
 
     log.info('Test dsidm role subtree-status')
+    topology_st.logcap.flush()
     subtree_status(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    result = topology_st.logcap.get_raw_outputs()
     check_value_in_log_and_reset(topology_st, check_value=output)
 
+    log.info('Test dsidm role subtree-status - json')
+    args.json = True
+    subtree_status(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    result = topology_st.logcap.get_raw_outputs()
+    json_result = json.loads(result[0])
+    assert json_result['entries'][0]['dn'] == test_role.dn
```

Comment:
I see the test fails here:

```
>       assert json_result['entries'][0]['dn'] == test_role.dn
E       AssertionError: assert 'cn=nsManaged...xample,dc=com' == 'cn=test_mana...xample,dc=com'
E         - cn=test_managed_role,dc=example,dc=com
E         ?    ^^ ^^^      ^^
E         + cn=nsManagedDisabledRole,dc=example,dc=com
E         ?    ^ ^      ^^^^^^^^^
```

Reply:
Odd, I'll look into this...  But I think I was just testing the function and not the entire file - so other roles got added from previous tests.  Anyway I'll fix it.